### PR TITLE
MRIID-54 Sidebar Cypress tests

### PR DESCRIPTION
**Pull request by fernandodonaire**
Added Cypress tests for the Sidebar component. These can be run by: 

1) Installing dependencies using: `npm install`.
2) Running the app locally using: `npm start`
3) Running Cypress using: `npx cypress open`
4) On the browser window that is opened, clicking on the `sidebar.js` integration to run it.  

Screen Shot 2021-02-18 at 12 49 47 PM

### CODE REVIEW

File: cypress/integration/sidebar.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+describe("Tests for the Sidebar component", () => {
+  beforeEach(() => {
+    cy.visit("localhost:3000");
+    cy.waitForReact();
+  });
```
Comment (SpencerMarks-IDHA) [inline cypress/integration/sidebar.js:5]: what is a cy?

**Review by SpencerMarks-IDHA — APPROVED** [review]
maybe rename cy for readablity

File: cypress/integration/sidebar.js (hunk 1 above)
Comment (fernandodonaire) [reply cypress/integration/sidebar.js:5]: So `cy` just stands for Cypress. So when you call `cy.visit` you are calling the `visit` method of the Cypress library.  🙂

**Comment by fernandodonaire** [comment]
Thanks for the review @SpencerMarks-IDHA !

**Pull request merged**